tsconfig.json

```diff
@@ -3,23 +3,24 @@
 	"compilerOptions": {
 		"alwaysStrict": true,
 		"lib": ["esnext"],
+		"strict": true,
 		"module": "commonjs",
 		"noUnusedParameters": true,
 		"outDir": "./dist",
 		"sourceMap": true,
-		"types": ["node"],
-        "declaration": true,
-        "noUnusedLocals": true,
-        "removeComments": false,
-        "target": "esnext"
+		"types": ["node", "discord.js", "klasa"],
```

Comment:
```suggestion
```

This *should* not be needed. TypeScript *should* automatically resolve types in node_modules

Reply:
Should, but it doesn't. This has always fixed many types errors, otherwise you'd have to import the modules in SOME file at least once.